app/index.js

```diff
@@ -482,7 +482,7 @@ app.on('ready', () => {
         title: 'Save password?',
         message: message,
         buttons: ['Yes', 'No'],
-        defaultId: 0,
+        defaultId: 1,
```

Comment:
Stop expert bullshits.
I think, your job is doing your fucking job. You're sayin 'safer' browser, in fact. It doesn't.
If you using someone's computer for few minutes, going password manager, and taking a screenshot and upload it internet, easier than installing rootkits etc. I'm talking about asking windows/osx password before showing saved fucking passwords.